### ISSUE TITLE
Validation on swpc_products branch  

### ISSUE THREAD

**Issue by ZhuxiaoLi66** [NOAA-SWPC/GSMWAM-IPE#39]
The new features have been added to SWIO, includes output of O/N2. 
The output of O/N2 from SWIO is verified by the calculation of the O/N2 offline based on other output variables (Temp., height, O_den and N2_den) from the same run by the corresponding NCL code. 
The NetCDF format of the output and other output variables also need to be double-checked aiming for a more ready-to-go status.

**Comment by ZhuxiaoLi66** [comment]
Hourly output of SWIO for a 6hrs 2013031600 run has been conducted, the new feature O/N2 output has been verified by the method introduced above.  The results look good, which verified Raffaele's new code work.

**Comment by ZhuxiaoLi66** [comment]
![O_to_N2_SWIO_20130316_01UT-06UT](https://user-images.githubusercontent.com/22546571/86956806-ee51c900-c116-11ea-99c2-8db914db4fac.png)
![O_to_N2_SWIO_NCL_20130316_01UT-06UT](https://user-images.githubusercontent.com/22546571/86956830-f7db3100-c116-11ea-9370-b9757611d080.png)
![O_N2_diff_SWIO_20130316_1UT-6UT](https://user-images.githubusercontent.com/22546571/86956840-ff9ad580-c116-11ea-88c9-bd9b0aa9997d.png)

**Comment by ZhuxiaoLi66** [comment]
As for the other issues about the NetCDF format of the output, there is a discussion on email base, will update after that.

**Comment by ZhuxiaoLi66** [comment]
I would like to update the discussion results at today's morning meeting in the aspect of gsm output dataset improvement. 
a) the missing (fill_in) value at the pole points of gsm*nc will be set into a parameter, which will have a value like -9999.
b) As for the issue of adding dimension variables to the goal variables & make 'time' an unlimited dimension variable. 
    Thanks for Raffaele's nice explain on the difficulty & risk on these due to SWIO code structure design from the beginning. 
    Also thanks for Gorge's introduction about the output format of the operational modeling in SWPC. 
    In short, we don't need to think about this anymore. 
c). I suggested Raffaele add the lev(x03) variable with long name ('pressure level') to the variable list of gsm*nc and change the variable name of 'wam_height_levels' back to 'height', would like to ask group's opinion on this.  thanks.

**Comment by ZhuxiaoLi66** [comment]
The following is the validation on another new variable iss_mass_density (neutral total mass density at 400km). I compared the direct output from SWIO and the one based on the interpolation from WAM pressure level output. the results look very good, the run is the same run to verify O/N2 yesterday. 

![DEN_400km_SWIO_20130316_1UT_6UT](https://user-images.githubusercontent.com/22546571/87091803-832bf380-c1f7-11ea-89d3-809ed7bc5d06.png)
![DEN_400km_WAM_20130316_1UT_6UT](https://user-images.githubusercontent.com/22546571/87091811-87581100-c1f7-11ea-80e1-bc7d647cf0e7.png)

**Comment by twfang** [comment]
The results look good. @ZhuxiaoLi66 once you and Raffaele finalize the code, we will get it merged.

**Comment by ZhuxiaoLi66** [comment]
Come to the conclusion, after Raffaele update the data format of gsm*nc, it should be fine to merge the current branch into the develop branch in WAM-IPE repository from my side. Good job, Raffaele.

**Comment by ZhuxiaoLi66** [comment]
@twfang you are right.
